samples/display/lvgl/sample.yaml

```diff
@@ -30,3 +30,12 @@ tests:
     platform_allow: nrf52840dk_nrf52840
     extra_args: SHIELD=buydisplay_2_8_tft_touch_arduino
     tags: shield
+  sample.display.dummy:
+    build_only: true
+    platform_allow: native_posix
+    extra_configs:
+      - CONFIG_DUMMY_DISPLAY=y
+      - CONFIG_DUMMY_DISPLAY_DEV_NAME="DISPLAY"
+      - CONFIG_KSCAN=n
+      - CONFIG_SDL_DISPLAY=n
```

Comment:
why is this different from `samples/display/boards/native_posix.conf`?

Reply:
It removes the requirement to have libsdl2 installed by using the dummy display driver instead.

Reply:
Will you please add this to the commit message?

Reply:
done